core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -435,6 +436,20 @@ class LogManager(logDirs: Seq[File],
       cleaner.startup()
   }
 
+  /**
+   * wait all jobs to complete
+   * @param jobs jobs
+   * @return true if all pass. Otherwise, false
+   */
+  private[log] def allPass(jobs: Seq[Future[_]]): Boolean = {
```

Comment:
The name makes this look generic, but it's not. The logging is very specific. I would change the name to be specific.

Reply:
How about "waitForAllToComplete"

Reply:
The log message talks explicitly about LogManager shutdown. Seems like you'd want it to say something about log manager shutdown. I would even set the type parameter to the type we expect from the shutdown method.

Reply:
> I would even set the type parameter to the type we expect from the shutdown method.

There is no explicit type so I didn't define the type parameter.